modules/default.nix

```diff
@@ -21,7 +21,7 @@ let
     in
       fold f res res.config.warnings;
 
-  extendedLib = import ./lib/stdlib-extended.nix lib;
+  extendedLib = import ./lib/stdlib-extended.nix extraSpecialArgs.lib or lib;
```

Comment:
This seems redundant with the line above, since `lib` is already equal to `extraSpecialArgs.lib or lib`.

Reply:
Only if the default were used. Not if another lib was explicitly passed as an argument. Yet the lib in the `extraSpecialArgs` would overwrite it in the end. To prevent having to extend it again on line 37, I did this.